packages/integration-tests/src/entities/BookCodegen.ts

```diff
@@ -67,6 +67,8 @@ export interface BookOrder {
   author?: AuthorOrder;
 }
 
+export const BookDefaultValues = { order: 0 };
```

Comment:
Nit picky but lower case `bookDefaultValues`?

Reply:
Will do, I wasn't sure what the convention here might be.